artifactory/commands/generic/deleteprops.go

```diff
@@ -31,15 +31,19 @@ func (deleteProps *DeletePropsCommand) Run() error {
 		return err
 	}
 
-	resultItems, searchErr := searchItems(deleteProps.Spec(), servicesManager)
-
-	propsParams := GetPropsParams(resultItems, deleteProps.props)
+	reader, searchErr := searchItems(deleteProps.Spec(), servicesManager)
+	if searchErr != nil {
+		return searchErr
+	}
+	defer reader.Close()
+	propsParams := GetPropsParams(reader, deleteProps.props)
 	success, err := servicesManager.DeleteProps(propsParams)
```

Comment:
This error is unchecked.

Reply:
We return it eventually but not right away, we want to fill the Results() for the print summary.